CPS-0THR/README.md

```diff
@@ -0,0 +1,137 @@
+---
+CPS: 0THR
+Title: Greater Transaction Throughput
+Category: Consensus
+Status: Open
+Authors:
+  - Arnaud Bailly <arnaud.bailly@iohk.io>
+  - Brian W. Bush <brian.bush@iohk.io>
+  - Hans Lahe <hans.lahe@iohk.io>
+Implementors: Intersect
+Discussions: 
+Created: 2024-10-11
+License: Apache-2.0
+---
+
+# CIP-???? Greater Transaction Throughput
+
+
+## Abstract
+
+The Cardano mainnet occasionally experiences congestion where there are too many transactions in the memory pool to be included in the next block or in the next few blocks. Sometimes the block utilization peaks above 90% for an extended period of time. This not only impacts general user experience but it can also severely impact use cases such as airdrops, oracles, partner chains, DEXes, and Dapps. Emerging use cases and application deployments promise to accelerate the need for high throughput on Cardano. Applied research on several fronts is needed to propose and provide evidence for techniques that increase throughput measured in terms of transactions, transaction size, and script execution units. Such work should be based on a clear understanding of stakeholder requirements.
+
+## Motivation
```

Comment:
```suggestion
```
please tweak the structure to align with the template
https://github.com/cardano-foundation/CIPs/blob/master/.github/CPS-TEMPLATE.md

Reply:
@bwbush it could also be easier to re-work by following the content outline indicated in https://github.com/cardano-foundation/CIPs/tree/master/CIP-9999 - especially in the case that this CPS might have been written as a CIP originally (to help sift the content of this draft between the CPS & its associate CIP).  Then the required outline will also conform to the template as a matter of course.

Reply:
Thanks! Fixed in 3f930b9ecbc80561e75deb89434773bbaa3ef589.